manifests/params.pp

```diff
@@ -1,9 +1,10 @@
 # Set default parameters based on distribution release
 class nfs::params {
   $portmap = $::operatingsystem? {
-    'Debian' => $::lsbdistcodename? {
-      'Wheezy' => 'rpcbind',
-      default  => 'portmap',
+    'Debian' => versioncmp($::operatingsystemmajrelease, 7)? {
+      1    => 'rpcbind',
```

Comment:
> You should quote numbers because future parser does not convert strings to integer and thus it will fail on puppet4

Good catch, thanks. I've done this and pushed to the proposed branch.